### DISCUSSION TITLE
mini.comment on cuda files

### DISCUSSION THREAD

**Discussion by twmht** [LazyVim/LazyVim#3155] [Q&A]
When I'm editing a CUDA file (.cu extension), mini.comment says 'Option 'commentstring' is empty'. How can I modify it to follow C++'s comment style?

**Comment by twmht** [comment]
I just found a solution and put it in options.lua

```lua
vim.api.nvim_create_autocmd("FileType", {
	pattern = "cuda",
	callback = function()
		vim.bo.commentstring = "// %s"
	end,
})

```
 Is there a better way to do this?

**Reply by yusufaktepe** [reply]
> ...
> Create a file 'after/ftplugin/blueprint.lua' (or whichever 'filetype' is actually set in your case) and add line `vim.bo.commentstring = '// %s'` to it.

https://github.com/echasnovski/mini.nvim/issues/351
